executor/load_data.go

```diff
@@ -211,6 +216,24 @@ func (e *LoadDataInfo) initLoadColumns(columnNames []string) error {
 	return nil
 }
 
+// initColAssignExprs creates the column assignment expressions using session context.
+// RewriteAstExpr will write ast node in place(due to xxNode.Accept), but it doesn't change node content,
+// so we sync it.
```

Comment:
what does `we sync it` mean?

Reply:
I don't know either. I copy this comment and some code from https://github.com/pingcap/tidb/pull/43492.